File: Perl/scripts/Problem001.pl
$start  = time();
$ans = 0;

for($i = 0; $i < 1000; $i += 1){
    if( $i%3==0 or $i%5==0 ){
        $ans += $i;
    }
}

$exc_time = time() - $start;
print "\nAnswer: $ans\n";
print "Time Taken: $exc_time\n\n";

File: Python/scripts/Problem001.py
from time import time


main = lambda: sum([i for i in range(1000) if (i % 3 == 0 or i % 5 == 0)])


if __name__ == '__main__':
    start = time()
    print(f"\nAnswer: { main() }")
    print(f"Time Taken: { time() - start }\n")
